ephemeralkey/client_test.go

```diff
@@ -17,7 +17,7 @@ func TestEphemeralKeyDel(t *testing.T) {
 func TestEphemeralKeyNew(t *testing.T) {
 	key, err := New(&stripe.EphemeralKeyParams{
 		Customer:      "cus_123",
-		StripeVersion: "2017-05-25",
+		StripeVersion: "2018-02-06",
```

Comment:
Figured I should bump this to the latest API version. Plan and Product updates reflect these changes with the removal of `name` and `statement_descriptor` from `Plan`.